Mathlib/Probability/Moments/SubGaussian.lean

```diff
@@ -545,6 +545,37 @@ lemma add_of_indepFun {Y : Ω → ℝ} {cX cY : ℝ≥0} (hX : HasSubgaussianMGF
       · exact hY.mgf_le t
     _ = exp ((cX + cY) * t ^ 2 / 2) := by rw [← exp_add]; congr; ring
 
+lemma add_of_HasSubgaussianMGF {Y : Ω → ℝ} {cX cY : ℝ≥0} (hcX : cX > 0) (hcY : cY > 0)
```

Comment:
The name of that lemma could simply be `HasSubgaussianMGF.add`. `X.add` is the standard name for a lemma that states that adding two `X` gives an `X`.
```suggestion
protected lemma add {Y : Ω → ℝ} {cX cY : ℝ≥0} (hcX : cX > 0) (hcY : cY > 0)
```
Can you move this lemma above `add_of_indepFun`? Right now it is in the middle of two lemmas about adding independent random variables.